### ISSUE TITLE
Add a mechanism for loading unknown file types

### ISSUE THREAD

**Issue by pont-us** [pont-us/PuffinPlot#124]
That is to say, some means whereby a user can specify column names or numbers for the various fields that PuffinPlot makes use of. Interactive dialog? Textual file specification? How and where to save the format spec?

**Comment by pont-us** [comment]
```
Bugs Everywhere data:
Created at: 2009-08-24 00:04:04
Status: fixed
Severity: minor
UUID: 99d32571-a0f5-430d-9074-9fdb22d0c1db
Short name: a88/99d
Ditz bug index: 122
```

**Comment by pont-us** [comment]
2012-05-20 09:48:10

This should actually have been closed on 6 February along with bug #121 (improve file import capabilities).

UUID: 802b7d51-be68-41bd-bc52-54e4e7f13f03
